setup.py

```diff
@@ -22,7 +22,11 @@
     install_reqs += ['argparse']
     tests_reqs += ['unittest2']
 
-readme = open('README.rst').read()
+if sys.version_info[0] == 3:
```

Comment:
Better check for `>2`. Assume that a future Python 4 is compatible unless stated otherwise.